flask_blogging/processor.py

```diff
@@ -71,6 +77,7 @@ def process(cls, post, render=True):
         post["priority"] = 0.8
         if render:
             cls.render_text(post)
+        post["meta"]["images"] = cls.extract_images(post)
```

Comment:
This should probably be in the `if render` section. That is why the tests are failing.